titan/model.py

```diff
@@ -387,6 +387,7 @@ def exit(self):
                     # agent ages out of model
                     if agent.age > exit.age:
                         self.exits[strategy.exit_class].append(agent)
+                        break
```

Comment:
Sorry this is on a somewhat relevant row, but not where I wanted it to be? Github won't let me choose a more obvious row.

Anyway! This whole `if case` feels very much a like switch statement structure! 

https://docs.python.org/3.10/whatsnew/3.10.html#pep-634-structural-pattern-matching

Reply:
We thought about switch statements, but we want to stay compatible with 3.9, which doesn't allow switch statements 😞